app/controllers/supplejack_api/records_controller.rb

```diff
@@ -87,6 +98,12 @@ def self.record_serializer_class
 
     private
 
+    def mlt_fields
+      return [] unless params[:fields]
+
+      params[:fields].split(',').map(&:to_sym)
```

Comment:
I dont remember how list params are passed to SJ

Reply:
I'm just having a look now, they might be available in the options[:fields]. That's how we are using them in the records_controller#index. You can see on the DetermineAvailableFields service.

Reply:
@Reinkaos Found it its handled similarly `@field_list = options[:fields].split(',').map { |f| f.strip.tr(':', '_').to_sym }` in search.

Reply:
Thanks for checking that!